client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -44,6 +44,12 @@ export class DisplayTable extends React.Component {
       (supplied_column_config) => ({
         ...column_config_defaults,
         ...supplied_column_config,
+        // Set visibility_toggleable default based off index
```

Comment:
No need to check if it's defined. Also no need for ternary here

```
    const col_configs_with_defaults = _.mapValues(
      column_configs,	      column_configs,
      (supplied_column_config) => ({	      (supplied_column_config) => ({
        // Set visibility_toggleable default based off index
        visibility_toggleable: supplied_column_config.index !== 0
        ...column_config_defaults,	        ...column_config_defaults,
        ...supplied_column_config,	        ...supplied_column_config,
      })
    );
```
So if `visibility_toggleable` was manually passed in, object spread operator ` ...supplied_column_config` will override the value.